cmd/prometheus-ethermine-exporter/main.go

```diff
@@ -393,20 +442,17 @@ func buildMinerRegistry(response http.ResponseWriter, pool *Pool, minerAddress s
-	util.NewGauge(registry, namespace, "miner", "income_usd", "Mined coins per second (converted to USD).", constLabels).Set(statsData.Data.USDPerMinute / 60)
-	util.NewGauge(registry, namespace, "miner", "income_btc", "Mined coins per second (converted to BTC).", constLabels).Set(statsData.Data.BTCPerMinute / 60)
-	// Deprecated
-	util.NewGauge(registry, namespace, "miner", "income_minute_coins", "(Deprecated) Mined coins per minute.", constLabelsWithCurrency).Set(statsData.Data.CoinsPerMinute)
-	util.NewGauge(registry, namespace, "miner", "income_minute_usd", "(Deprecated) Mined coins per minute (converted to USD).", constLabels).Set(statsData.Data.USDPerMinute)
-	util.NewGauge(registry, namespace, "miner", "income_minute_btc", "(Deprecated) Mined coins per minute (converted to BTC).", constLabels).Set(statsData.Data.BTCPerMinute)
+	util.NewGauge(registry, namespace, "miner", "income_minute_coins", "Estimated number of coins (in base units) mined per minute (based on your average hashrate as well as the average block time and difficulty of the network over the last 24 hours.).", constLabelsWithCurrency).Set(statsData.Data.CoinsPerMinute)
+	util.NewGauge(registry, namespace, "miner", "income_minute_usd", "Estimated number of USD (in base units) mined per minute (based on your average hashrate as well as the average block time and difficulty of the network over the last 24 hours.).", constLabels).Set(statsData.Data.USDPerMinute)
+	util.NewGauge(registry, namespace, "miner", "income_minute_btc", "Estimated number of BTC (in base units) mined per minute (based on your average hashrate as well as the average block time and difficulty of the network over the last 24 hours.).", constLabels).Set(statsData.Data.BTCPerMinute)
 
 	// Worker stats
 	workerLabels := make(prometheus.Labels)
 	workerLabels["worker"] = ""
-	workerLastSeenMetric := util.NewGaugeVec(registry, namespace, "worker", "last_seen_seconds", "Delta between time of last statistics entry and when the miner was last seen (s).", constLabels, workerLabels)
-	workerReportedHashRateMetric := util.NewGaugeVec(registry, namespace, "worker", "hashrate_reported_hps", "Current hash rate for a worker as reported from the worker (H/s).", constLabels, workerLabels)
-	workerCurrentHashRateMetric := util.NewGaugeVec(registry, namespace, "worker", "hashrate_current_hps", "Current hash rate for a worker (H/s).", constLabels, workerLabels)
+	workerLastSeenMetric := util.NewGaugeVec(registry, namespace, "worker", "last_seen_seconds", "Unix timestamp of when the worker was last seen by the pool.", constLabels, workerLabels)
+	workerReportedHashRateMetric := util.NewGaugeVec(registry, namespace, "worker", "hashrate_reported_hps", "Reported hashrate of the worker in H/s.", constLabels, workerLabels)
+	workerAverageHashRateMetric := util.NewGaugeVec(registry, namespace, "worker", "hashrate_average_hps", "Average hashrate of the worker in H/s during the last 24h.", constLabels, workerLabels)
```

Comment:
I'm a bit torn about adding aggregated metrics wrt. best practices, since this can already be calculated from the current rate, but if you think it might be useful then it's fine by me. But I'd prefer "24 hours" in the help text instead of "24h".